tests/plugins/webp-uploads/helpers/class-wp-image-doesnt-support-webp.php

```diff
@@ -2,7 +2,7 @@
 /**
  * A WP_Image_Editor mock that doesn't support WebP.
```

Comment:
Let's name this folder `data` instead of `helpers`, similar to how you did it for Dominant Color Images, and in accordance with both the original files being in the `testdata` folder.